packages/pwa-kit-create-app/scripts/trim-extensions.js

```diff
@@ -13,12 +13,14 @@ const path = require('path')
 const pluginConfig = require('../assets/plugin-config')
 const {execSync} = require('child_process')
 
-const removeComponentCandidates = [] // List of files that are candidates for removal, as a result of trimming.
+const removeComponentCandidates = new Set() // List of files that are candidates for removal, as a result of trimming.
 const SEPARATOR = path.sep // Use OS-specific path separator
 const COMPONENT_SCAN_PATHS = [
     `${SEPARATOR}src${SEPARATOR}components${SEPARATOR}`,
     `${SEPARATOR}src${SEPARATOR}pages${SEPARATOR}`,
-    `${SEPARATOR}src${SEPARATOR}hooks${SEPARATOR}`
+    `${SEPARATOR}src${SEPARATOR}hooks${SEPARATOR}`,
+    `${SEPARATOR}src${SEPARATOR}routes.tsx`,
+    `${SEPARATOR}config${SEPARATOR}constants.js`
```

Comment:
Added additional files as targets for trimming (it may grow when we have more extensions folded in)